### PR TITLE
692 investigate softnas backup

### PR DESCRIPTION
**Pull request by tomskelley-gjs**
Use *much* more recent AWS provided docker container.

### CODE REVIEW

File: charts/nfs-backup/templates/cronjob.yml [hunk 1]
```diff
@@ -50,4 +95,4 @@ spec:
               claimName: nfs-homes
           - name: exclude-patterns
             configMap:
-              name: {{ template "fullname" . }}
+              name: {{ template "fullname" . }}
```
Comment (damacus) [inline charts/nfs-backup/templates/cronjob.yml:98]: why no EOL character?

Comment (tomskelley-gjs) [reply charts/nfs-backup/templates/cronjob.yml:98]: Doesn't need one. SSDs aren't cheap!

File: charts/nfs-backup/values.yaml [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
-  IAMRole: ""
+  IAMRole: "alpha_nfs_backup"
   S3:
-    BucketName: ""
+    BucketName: "alpha-moj-analytics-nfs-backup"
 Image:
-  Repository: samepagelabs/s3cmd
-  Tag: "1.6.1"
+  Repository: amazon/aws-cli
+  Tag: "latest"
```
[the remark this answers is not in the document]
Comment (tomskelley-gjs) [reply charts/nfs-backup/values.yaml:7]: Pinned

File: charts/nfs-backup/templates/cronjob.yml (hunk 1 above)
Comment (tomskelley-gjs) [reply charts/nfs-backup/templates/cronjob.yml:98]: Fixed just in case it breaks any automation

**Pull request merged**